.github/workflows/azops.yml

```diff
@@ -3,88 +3,90 @@ name: AzOps
 on:
   repository_dispatch:
     types:
-      - "activity-logs"
+      - "Azure Activity Logs"
+      - "GitHub CLI"
+      - "PowerShell"
+      - "Bash"
   pull_request:
     branches:
-      - main
+      - "main"
     paths:
       - "azops/**"
   pull_request_review:
     branches:
-      - main
-    types: [submitted]
+      - "main"
+    types:
+      - "submitted"
     paths:
       - "azops/**"
 
+env:
+  AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+  AZOPS_STATE: "azops"
+  AZOPS_DEFAULT_DEPLOYMENT_REGION: "northeurope"
+  AZOPS_ENROLLMENT_ACCOUNT: ""
+  AZOPS_OFFER_TYPE: "MS-AZR-0017P"
```

Comment:
Not really relevant to this PR, but this got me thinking... do we plan to support provisioning "Enterprise Dev/Test" (Offer MS-AZR-0148P) Subscriptions? If so, we would need to add some logic to allow this in AzOps.

Reply:
Let's take this offline and talk through it.